pennylane_lightning/core/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_Param.cpp

```diff
@@ -958,6 +958,41 @@ TEMPLATE_TEST_CASE("StateVectorKokkos::applyOperation param "
             CHECK(real(sv_gate_host[j]) == Approx(real(expected_result[j])));
         }
     }
+
+    SECTION("2-controlled SingleExcitationMinus") {
+        Kokkos::deep_copy(sv_gate.getView(), ini_sv);
+
+        const std::vector<std::size_t> control_wires = {0, 2};
+        const std::vector<bool> control_values = {true, false};
```

Comment:
Could you also check with contiguous controlled wires, for example `control_wires={0, 1, 2}` ?

Reply:
test added